mypy.ini

```diff
@@ -1,4 +1,6 @@
 [mypy]
+files = pyteal,scripts,tests
```

Comment:
Should we remove the `MYPY` variable in the `Makefile` as well, now that the config is taking over this decision?

Reply:
Yes, that makes sense. Done ✅